docs-src/clusters/how-to-set-up-sqlite-visibility-store.md

```diff
@@ -0,0 +1,65 @@
+---
+id: how-to-set-up-sqlite-visibility-store
+title: How to set up SQLite Visibility store
+sidebar_label: SQLite
+description: You can set SQLite as your Visibility store with any other supported Persistence databases.
+tags:
+  - operation-guide
+  - filtered-lists
+  - visibility
+ssdi:
+  - SQLite v3.31.0 and later
+---
+
+You can set SQLite as your [Visibility store](/concepts/what-is-visibility).
+Verify [supported versions](/clusters/how-to-set-up-visibility-in-a-temporal-cluster#supported-databases) before you proceed.
+
+Temporal only support in-memory database with SQLite; this means that the database is automatically created when Temporal Server starts and is destroyed when Temporal Server stops.
+
+You can change the configuration to use file-based database so that it is preserved when Temporal Server stops. However, note that if using a file-based SQLite database, upgrading your database schema to enable Advanced Visibility features is not supported; in this case, you will have to delete the database and create it again to upgrade.
```

Comment:
```suggestion
You can change the configuration to use a file-based database so that it is preserved when Temporal Server stops.
However, if you use a file-based SQLite database, upgrading your database schema to enable Advanced Visibility features is not supported; in this case, you must delete the database and create it again to upgrade.
```